src/fsharp/NicePrint.fs

```diff
@@ -1275,7 +1277,7 @@ module InfoMemberPrinting =
         let paramDatas = minfo.GetParamDatas(amap, m, minst)
         let layout =
             layout ^^
-                if isNil (List.concat paramDatas) then
+                if Seq.isEmpty (Seq.concat paramDatas) then
```

Comment:
Please don't introduce Seq in the compiler unless there is a specific perf measurement, for much the same reasons as mentioned in other PR.

Reply:
In this case it's not because of the intermediate allocation, but because it may will exit on very first item in paramDatas instead of concat and then throwing it away. But I can rewrite that as List.exists (not isNil) paramDatas

Reply:
In this case it's not because of the intermediate allocation, but because it may will exit on very first item in paramDatas instead of concat and then throwing it away. But I can rewrite that as List.exists (not isNil) paramDatas